modules/vpc-sc/variables.tf

```diff
@@ -15,7 +15,7 @@
  */
 
 variable "access_levels" {
-  description = "Access Levels."
+  description = "Map of Access Levels to be created. For each Access Level you can specify 'ip_subnetworks, required_access_levels, members, nagete, device_policy or regions'."
```

Comment:
typo on `nagete`

Reply:
Should you add the extra fields like `device_policy` to the condition object type?